sunpy/instr/goes.py

```diff
@@ -138,7 +230,7 @@ def temp_em(goeslc, abundances="coronal", download=False, download_dir=DATA_PATH
     2014-01-01 00:00:02  7e-07  7e-06
     2014-01-01 00:00:04  7e-07  7e-06
     2014-01-01 00:00:06  7e-07  7e-06
-    >>> goeslc_new = temp_em(goeslc)
+    >>> goeslc_new = calculate_temperature_em(goeslc)
```

Comment:
Also, I don't thing calling a variable `..._new` is a good practice that we should encourage.  Probably we can find a better name for it.